docs/datacite_integration.md

```diff
@@ -0,0 +1,59 @@
+---
+title: DataCite Integration
+tags:
+  - DataCite
+  - DOI
+  - Fragaria
+---
+
+# DataCite Integration
+
+Archipelago's 1.6.0+ release (December 2025) features DataCite Integration, including UI interfaces, DataCite Schema V4 compliance, stable reporting, and test and production environment. Archipelago's DataCite functionality is integrated into the Fragaria Module and RAW JSON workflows.
+
+This documentation page is a work-in-progress and will be updated in the upcoming weeks to provide more detailed information related to this functionality area in the upcoming weeks.
+
+??? note "DataCite Fabrica Account Required"
+
+    To use Archipelago's DataCity Integration, your institution will need to have established DataCite Fabrica account credentials. Archipelago will not provide test account credentials for you to use. 
+    
+    You can find more information about establshing a DataCite Fabrica account at: https://doi.datacite.org.
+
+
+### Configuration Form
+
+You can find the DataCite DOI Integration Settings Configuration Form:
+
+- Through the `Manage` menu > `Configuration` > `Archipelago` > `DataCite Services`
+- Directly at `/admin/config/archipelago/fragaria/DataCite` 
+
+![DataCite Config Form](images/DataCite_config_form.png)
+
+Please note that your account credentials will be not show the saved password after your initial configuration form save.
+
+### Corresponding Metadata Display Template
+
+The default Archipelago 1.6.0+ release includes a custom Twig template with DataCite V4 mappings for default Archipelago instances. This default template includes all required DataCite elements, plus a few optional elements.
+
+For default Archipelago instances, you can find this template at ~`/metadatadisplay/20`.
```

Comment:
@DiegoPino --noting that I will check on this when the final release configs are pushed next week